### PR TITLE
CS-40 cookie consent plugin customization

### PR DESCRIPTION
**Pull request by zsso**

### CODE REVIEW

File: csatar-theme/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/node_modules
+/vendor
+composer.lock
+package-lock.json
```
Comment (hupupu) [inline csatar-theme/.gitignore:4]: Lock files are intended to be commited to source control. It is the way to make sure everyone has the same packages

File: csatar-theme/partials/site/meta.htm [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<meta charset="utf-8">
+<title>{{ 'CSATÁR'|_ }} - {{ placeholder('pageTitle') ?: (this.page.meta_title ?: this.page.title) }}</title>
```
Comment (hupupu) [inline csatar-theme/partials/site/meta.htm:2]:
```suggestion
<title>{{ placeholder('pageTitle') ?: (this.page.meta_title ?: this.page.title) }} - {{ 'CSATÁR'|_ }}</title>
```

File: csatar-theme/layouts/default.htm [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        <!-- Content -->
+        <section id="layout-content" class="flex-grow-1">
+            {% page %}
+        </section>
+
+        <!-- Footer -->
+        <footer id="layout-footer">
+            {% partial 'site/footer' %}
+            {% component 'cookiebannerOverRide' %}
+        </footer>
```
Comment (hupupu) [inline csatar-theme/layouts/default.htm:28]: The readme of the theme states:

**Important**: Make sure you keep the `{% styles %}` and `{% scripts %}` placeholder tags as these are used by plugins for injecting assets.

**Review by hupupu — CHANGES_REQUESTED** [review]
1. De we really need `csatar-theme/assets/images/theme-preview_kept_for_reference.png`?
2. Shouldn't we use jQuery, bootstrap and fontawesome from CDN (and not keep it in the repo)?
3. If I remember well, we agreed that we will use Vue, if we do, including jQuery is a bad idea
4. You don't have a trailing newline in any of the files. Even the pull request complains about it.

File: csatar-theme/.gitignore (hunk 1 above)
Comment (zsso) [reply csatar-theme/.gitignore:4]: 1. I wanted to keep it until we create a task for Kincső, to have a them-preview file with right dimensions (if wee need it)
2. This is the way jQuery and bootstrap was included to Demo theme, I just upgraded bootstrap.
3. I remember that we talked about it, but I don't remember there was a final decision.
4. Added
5. Removed those files from .gitignore (were added by default in Demo theme)

File: csatar-theme/layouts/default.htm (hunk 3 above)
Comment (zsso) [reply csatar-theme/layouts/default.htm:28]: Those tags are in the meta.htm partial.

File: csatar-theme/partials/site/meta.htm (hunk 2 above)
Comment (zsso) [reply csatar-theme/partials/site/meta.htm:2]: Changed.

**Pull request merged**